rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -931,6 +944,18 @@ def _update_participant_summary(
 
         dna_program_consent_update_code = config.getSettingJson(config.DNA_PROGRAM_CONSENT_UPDATE_CODE, None)
 
+        if 'verified' and 'verified_on' in remote_id_info:
```

Comment:
I know there was a comment to change it to this, but the logic here doesn't look like what was meant. This `if` condition is evaluating to True if `'verified_on' in remote_id_info` is true. This is not checking if 'verified' and 'verified_on' are in `remote_id_info`. Instead it's testing if the 'verified' string is truthy (which will always be True since non-empty strings are truthy).

Reply:
Nice catch, Thanks!